src/components/Icon/hooks/useIconProps.js

```diff
@@ -34,8 +35,10 @@ export default function useIconProps({
   }, []);
 
   const computedClassName = useMemo(() => {
-    return cx("icon_component", className, {
+    return cx(styles.iconComponent, "icon_component", className, {
+      [styles.iconComponentClickable]: clickable,
```

Comment:
I think here the prefix is not needed but it's not a blocker (clickable was a bit more readable than iconComponentClickable because we already have icon component class name as wrapper)